client/src/App.js

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './app.css';
+import { Dashboard, Home } from './layouts';
+import { LoginPage, RegisterPage, LoginGymPage, GymPage, NotfoundPage } from './pages';
 
 function App() {
   return (
-    <div className="continer">
-      <h1>Welcome To FitHouse</h1>
-      <h2> Hi</h2>
-    </div>
+    <Routes>
+      {/* Routes For Site Views {login, gym filter,...} */}
+      <Route path="/" element={<Home />}>
+        <Route index element={<LoginPage />} />
+        <Route path="login" element={<LoginPage />} />
+        <Route path="register" element={<RegisterPage />} />
+        <Route path="gym/login" element={<LoginGymPage />} />
```

Comment:
we still have a search route and gym profile
Add them, and give all your pages a container className